### PR TITLE
Feature/submit button spinner

### PR DESCRIPTION
**Pull request by thomasvanophem-betty**

### CODE REVIEW

**Comment by jeroenreijs** [comment]
Please never merge the edge branch into yours. This could mean that you have something in your branch that we don't want in acceptance (yet). See this commit: https://github.com/bettyblocks/material-ui-component-set/pull/303/commits/981b2ae998d0f929d711e5ee5d1f9649fa0e3377

**Comment by thomasvanophem-betty** [comment]
Hmm.. didn't think that one completely through.. I saw a merge conflict and thought.. let's fix it 😅  Will fix and won't happen again.

File: src/components/button.js [hunk 1]
```diff
@@ -22,8 +22,8 @@
       actionId,
       buttonText,
     } = options;
-
-    const { env, useText } = B;
+    const { loading: parentLoading } = parent || {};
```
Comment (vdberganton) [inline src/components/button.js:25]: Why do this with prop drilling if you can achieve this with a triggerEvent?

Comment (thomasvanophem-betty) [reply src/components/button.js:25]: Because the form already passes the loading state to it's children. If you prefer a triggerEvent I can change that.

Comment (vdberganton) [reply src/components/button.js:25]: Yes please, that is probably before we had triggerEvent and needs to be changed.

**Pull request merged**